gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/KafkaJobStatusMonitor.java

```diff
@@ -246,6 +256,19 @@ protected void processMessage(DecodeableKafkaRecord<byte[],byte[]> message) {
     }
   }
 
+  private void clearStartDeadlineTriggerAndDagAction(String flowGroup, String flowName, String flowExecutionId, String jobName) {
+
```

Comment:
same as above let's log when this is used and javadoc for when it should be called

Reply:
also put todo for a metric